samples/bluetooth/path_loss_monitoring/Peripheral/prj.conf

```
@@ -0,0 +1,13 @@
+CONFIG_LOG=y
+
+CONFIG_DK_LIBRARY=y
+
+CONFIG_BT=y
+
+CONFIG_BT_PERIPHERAL=y
+
+CONFIG_BT_DEVICE_NAME="Path_Loss_Peripheral"
+
+CONFIG_BT_PATH_LOSS_MONITORING=y
```

Could we try just with the controller power control kconfig?
```suggestion
CONFIG_BT_CTLR_LE_POWER_CONTROL=y
```